src/schema/schema.graphql

```diff
@@ -201,8 +201,27 @@ type Instance {
 	start: Float
 	dueDate: Float
 	tzoDue: Int
-	isRecurring: Boolean
```

Comment:
Either this needs to stay in, or we should stop denormalizing it in the entity definition above

Reply:
Removed from normalisation logic